akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -7,6 +7,8 @@ project.description: Shard a clustered compute process across the network with l
 For the Akka Classic documentation of this feature see @ref:[Classic Cluster Sharding](../cluster-sharding.md)
 @@@
 
+@@project-info{ projectId="akka-cluster-sharding-typed" }
+
```

Comment:
missing in typed/cluster-singleton.md and typed/distributed-data.md

Reply:
Added.